src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -358,11 +368,11 @@ void func_809DFE98(Actor* thisx, PlayState* play) {
 s32 EnCow_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
     EnCow* this = (EnCow*)thisx;
 
-    if (limbIndex == 2) {
-        rot->y += this->someRot.y;
-        rot->x += this->someRot.x;
+    if (limbIndex == COW_LIMB_HEAD) {
+        rot->y += this->headTilt.y;
```

Comment:
I know it's from MM but
```suggestion
        rot->y += this->headRot.y;
```
is more consistent with how we name these rotation fields